include/deal.II/fe/fe_nothing.h

```diff
@@ -82,10 +82,12 @@ class FE_Nothing : public FiniteElement<dim,spacedim>
 public:
 
   /**
-   * Constructor. Argument denotes the number of components to give this
-   * finite element (default = 1).
+   * Constructor. First argument denotes the number of components to give this
+   * finite element (default = 1). Second argument decides whether FE_Nothing
+   * will dominate any other FE in compare_for_face_domination() (default = false).
    */
-  FE_Nothing (const unsigned int n_components = 1);
+  FE_Nothing (const unsigned int n_components = 1,
+              const bool dominate = false);
```

Comment:
I think if you want to go down this road, you will have to explain what this means. Namely, that at interfaces where, for example, a Q1 meets an `FE_Nothing`, you will force the traces of the two functions to be the same. Because the `FE_Nothing` encodes a space that can is zero everywhere, this means that you force the Q1 field to become zero at this interface. This is a significant change from what we currently do. Is this what you intend?